docs/home/howto/LOGGING.md

```diff
@@ -181,64 +186,66 @@ There is example of policy below. Be aware that this is only example, and it nee
 }
 ```
 
-Example above shows configuration with rollover daily or when index achieve 1 GB size. Indexes older than 14 days will
+Example above shows configuration with rollover index policy on a daily basis or when the index achieve 1 GB size. Indexes older than 14 days will
 be deleted. States and conditionals could be combined. Please
-see [policies](https://opendistro.github.io/for-elasticsearch-docs/docs/im/ism/policies/) documentation for more
+see [policies](https://opensearch.org/docs/latest/im-plugin/ism/policies//) documentation for more
```

Comment:
Additional `/` in the link results in redirection to latest docs instead of policies